users/views_test.py

```diff
@@ -118,3 +121,77 @@ def test_countries_states_view(client):
     assert len(countries.get("FR").get("states")) == 0
     assert countries.get("US").get("name") == "United States"
     assert countries.get("TW").get("name") == "Taiwan"
+
+
+def test_create_email_change_request_invalid_password(user_drf_client, user):
+    """Test that invalid password is returned"""
+    resp = user_drf_client.post(
+        "/api/change-emails/",
+        data={
+            "new_email": "abc@example.com",
+            "password": user.password,
+            "old_password": "abc",
+        },
+    )
+    assert resp.status_code == status.HTTP_400_BAD_REQUEST
+
+
+def test_create_email_change_request_existing_email(user_drf_client, user):
+    """Test that create change email request gives validation error for existing user email"""
+    new_user = UserFactory.create()
+    user_password = user.password
+    user.password = make_password(user.password)
+    user.save()
+    resp = user_drf_client.post(
+        "/api/change-emails/",
+        data={"new_email": new_user.email, "password": user_password},
+    )
+    assert resp.status_code == status.HTTP_400_BAD_REQUEST
+
+
+def test_create_email_change_request_valid_email(user_drf_client, user, mocker):
+    """Test that change request is created"""
+    user_password = user.password
+    user.password = make_password(user.password)
+    user.save()
+
+    mock_response = mocker.patch.object(requests.Session, "get")
+    mock_response.raise_for_status.return_value = None
```

Comment:
Remove this mock and instead mock the new task, you don't need to assert any behavior of the task call, since that will be done in `serializers_test.py`